README.txt

```diff
@@ -1,21 +1,24 @@
-Methodocracy
+Methodocracy (noun):
+A cultural-political ideology which proposes that optimal decision-making, from the individual to national level, is necessarily based upon the quantity and quality of data, interpretation, and conclusions gathered through tested and reliable methodologies.
 
-"The Super-Power of Problem-Solving"
 
```

Comment:
Although I said right now the target audience is different, this tagline is meant for the ultimate target audience. There is no use having a temporary tagline.